src/js/themes/hpe.js

```diff
@@ -874,141 +894,160 @@ export const hpe = deepFreeze({
   },
   heading: {
     color: 'text-strong',
+    weight: 450,
     level: {
       1: {
         small: {
-          size: '36px',
-          height: '42px',
+          size: '48px', // size: '36px',
+          height: '48px', // height: '42px',
           maxWidth: '854px',
         },
         medium: {
-          size: '53px',
-          height: '59px',
+          size: '72px', // size: '53px',
+          height: '72px', // height: '59px',
```

Comment:
Given that we're setting line height to 1em, we should test with heading text that wraps, to ensure that the multiple lines don't appear too squished.

Reply:
I'm cataloging a list of questions and components to evaluate in the main issue: https://github.com/grommet/hpe-design-system/issues/3020#issue-1448456130